app/screens/receive-bitcoin-screen/receive-btc.tsx

```diff
@@ -227,7 +228,10 @@ const ReceiveBtc = () => {
   )
   const { convertCurrencyAmount } = usePriceConversion()
 
-  const { data } = useReceiveBtcQuery({ fetchPolicy: "cache-first" })
+  const { data } = useReceiveBtcQuery({
+    fetchPolicy: "cache-first",
+    skip: !useIsAuthed(),
```

Comment:
Not critical but we should probably follow one pattern of using the hook. either ` const isAuth = useIsAuth()` before using the variable `isAuth` or using the hook directly like is used here. I prefer the later though.